lib/octokit/default.rb

```diff
@@ -43,11 +41,10 @@ module Default
     end
 
     class << self
-
       # Configuration options
       # @return [Hash]
       def options
-        Hash[Octokit::Configurable.keys.map{|key| [key, send(key)]}]
+        Hash[Octokit::Configurable.keys.map {|key| [key, send(key)]}]
```

Comment:
This could use `Array#to_h`, which Rubocop automatically introduced in a similar place above. I'm not sure why it didn't do it here.